charts/dsp-atlantis/values.yaml

```diff
@@ -20,6 +20,8 @@ azure:
 vault:
   azure:
     pathPrefix: /path/to/azure/secrets
+  github:
+    pathPrefix: /path/to/github/secrets
```

Comment:
Wondering if this should be null? Otherwise the `required` will never print out an error message. I do see that this differs from the Azure pattern, maybe @mflinn-broad can offer some thoughts!

Reply:
happy to entertain nits, that's how we fix things. I literally just copied the azure line above it without thinking about consequences. `required`, I would prefer to have validation explicitly catch. Maybe commentizing is a good middle ground? 

```
pathPrefix: #/path/to/github/secrets
```

Reply:
went again at made the change, not really a good reason to leave as is

Reply:
Yeah, sorry about that @jyang-broad. The azure line was me going fast and getting a little sloppy just to get azure stuff working. Didn't mean to trick you into a bad pattern. I'll fix that real quick.